cloud/rust/TokenConsumer.rs

```diff
@@ -0,0 +1,50 @@
+
+use futures::TryStreamExt;
+use pulsar::{Authentication, Consumer, ConsumerOptions, Pulsar, SubType, TokioExecutor};
+use pulsar::consumer::InitialPosition;
+
+#[tokio::main]
+async fn main() -> Result<(), pulsar::Error> {
+    env_logger::init();
+
+    let addr = "{{ SERVICE_URL }}".to_string();
+    let mut builder = Pulsar::builder(addr, TokioExecutor);
+    let token = "{{ AUTH_PARAMS_TOKE }}".to_string();
```

Comment:
```suggestion
    let token = "{{ AUTH_PARAMS_TOKEN }}".to_string();
```
There are several more occurrences that need to be corrected too:)